packages/platform/platform-fastify/src/services/PlatformFastifyResponse.ts

```
@@ -0,0 +1,95 @@
+import {PlatformContext, PlatformResponse} from "@tsed/common";
+import contentDisposition from "content-disposition";
+import {FastifyReply} from "fastify";
+import type {ServerResponse} from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)